compiler/damlc/tests/daml-test-files/TemplateTypeRep.daml

```diff
@@ -0,0 +1,40 @@
+-- Copyright (c) 2019, Digital Asset (Switzerland) GmbH and/or its affiliates.
```

Comment:
We should also add a test that templates in different packages but with the same name are considered different, i.e.,
```
templateTypeRep @A.T =/= templateTypeRep @B.T
```

Reply:
Do you mean different modules? Of course, they should also be different for different packages but that doesn’t really fit into this test suite and also doesn’t seem to match the example in your comment.

Reply:
Yes, I meant modules. My bad.